package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "magento-extend-integration",
+  "description": "this npm package is used to manage linting dependencies only",
+  "version": "1.0.0",
+  "repository": "git@github.com:helloextend/magento-extend-integration.git",
+  "author": "Miles McLeod <miles.mcleod@extend.com>",
```

Comment:
Didn't realize we were re-branding, you _have_ to be the spokesperson!

Reply:
Oh boy. Gotta remove this haha